app/exporters/odt/cv.rb

```diff
@@ -39,16 +43,21 @@ def insert_general_sections(report)
       report.add_field(:version, '1.0')
       report.add_field(:comment, 'Aktuelle Ausgabe')
     end
-    # rubocop:enable Metrics/AbcSize
 
+    # rubocop:enable Metrics/AbcSize
     # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/LineLength
     def insert_personalien(report)
       report.add_field(:title, person.title)
-      report.add_field(:birthdate, Date.parse(person.birthdate.to_s).strftime('%d.%m.%Y'))
-      report.add_field(:nationalities, nationalities)
-      report.add_field(:email, person.email)
-      report.add_image(:profile_picture, person.picture.path) if person.picture.file.present?
+      unless @anon
+        report.add_field(:birthdate, Date.parse(person.birthdate.to_s)
+                                         .strftime('%d.%m.%Y'))
+        report.add_field(:nationalities, nationalities)
+        report.add_field(:email, person.email)
+      end
+      report.add_image(:profile_picture, person.picture.path) unless @anon || person.picture.file.blank?
```

Comment:
You can put that into the block as well. Just change it back to `if person.picture.file.blank?`